src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public void setBall(String color, int number) {
+        this.color = color;
+        this.number = number;
```

Comment:
The method name `setBall` is not descriptive of what it does. A more conventional name would be `setProperties` or even better, use separate methods `setColor` and `setNumber` for setting individual properties.